homeassistant/components/acmeda/base.py

```diff
@@ -0,0 +1,60 @@
+"""Base class for Acmeda Roller Blinds."""
+import aiopulse
+
+from homeassistant.core import callback
+from homeassistant.helpers import entity
+
+from .const import DOMAIN, LOGGER
+
+
+class AcmedaBase(entity.Entity):
+    """Base representation of an Acmeda roller."""
+
+    def __init__(self, hass, roller: aiopulse.Roller):
```

Comment:
Don't pass in `hass`. It will be set on the entity when it has been added to home assistant.

Reply:
Removed